### DISCUSSION TITLE
Locked taskbar weirdness with toolbars

### DISCUSSION THREAD

**Discussion by jasmeralia** [valinet/ExplorerPatcher#2689] [General]
I have several folders that I have pinned to the taskbar to access common applications quickly and easily. I'm running into a weird issue where if I lock the taskbar, it shows part of the first icon, but if the taskbar is unlocked, it looks fine.

Unlocked:

![toolbar-unlocked](https://github.com/valinet/ExplorerPatcher/assets/185758/2373c318-8305-4835-8293-584c18335431)

Locked:

![toolbar-locked](https://github.com/valinet/ExplorerPatcher/assets/185758/36d486ca-2859-42ff-93c4-781026ed986c)

I'm using Windows 11 Pro Build 22621.ni_release.220506-1250 with ExplorerPatcher 22621.2861.62.2:

![2024-01-12_06-13-34](https://github.com/valinet/ExplorerPatcher/assets/185758/f07f3de3-b1e7-48f1-9c10-2e715aa462e7)

It's not really critical, but was wondering if there was a way to solve this, as leaving it unlocked, I accidentally move things around sometimes when I don't intend to.

**Comment by pyrates999** [comment]
That is a preview release: https://blogs.windows.com/windows-insider/2024/01/11/releasing-windows-11-builds-22621-3078-and-22631-3078-to-the-release-preview-channel/

Uninstall EP, reboot, and see if it still happens.

**Comment by jasmeralia** [comment]
That's strange, because I don't have previews turned on?

![2024-01-12_07-19-39](https://github.com/valinet/ExplorerPatcher/assets/185758/582ea5f7-7f7b-47b1-9164-6afde953c3ec)

I'm not in Insider, either (I made that mistake a long time ago, unenrolled, and had to wait many months for it to catch back up to the current GA release to be able to use EP again).

Any idea what I may have missed that put me on a preview version, and/or how to get off of it?

**Reply by pyrates999** [reply]
In the settings app, with "Get the latest updates as soon as available" unticked, preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

So when you see an update that is prompted to be installed, ignore it.
